promscale/installation/docker.md

```diff
@@ -93,17 +97,29 @@ You can also migrate to Debian version by doing the following (please note: this
      END LOOP;
    END$$;
    ```
+   This is necessary because the collation in the Alpine image is broken and so
+   BTREE-based indexes remain incorrect until they are reindexed. It is
+   extremely important to execute this step before ingesting new data to avoid
+   data corruption. This process can take a long time depending on the indexed
+   textual data in the database.  
+
+1. Restart the Promscale Connector
 
-7. Restart the Promscale Connector
+</procedure>
```

Comment:
This entire procedure ☝️ needs more actual commands in it.

Reply:
Hi @VineethReddy02  could you please provide the commands for all the steps in the procedure.

Reply:
I'm removing the upgrade section from install docs.